native/src/main/scala/slinky/native/TouchableHighlight.scala

```diff
@@ -0,0 +1,18 @@
+package slinky.native
+
+import slinky.core.ExternalComponent
+import slinky.core.annotations.react
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+@react object TouchableHighlight extends ExternalComponent {
+  case class Props(onPress: () => Unit,
```

Comment:
Looks like `onPress` is optional according to the docs, so this should be `js.UndefOr[() => Unit]`. Same for `TouchableOpacity`.

Reply:
That is true, but is it useful to have a Touchable component without an onPress action? Don't we lose some typesafety by making it optional?

Reply:
My general approach is to match what the docs say, and in this case the `onPress` is marked as optional, so I would prefer having the type here also be optional so that anything that works in JS also works in Slinky.

For typesafety, I feel that the more important aspect is the shape of the value when it is provided, not as much having the property passed in or not since the developer usually has an idea of which props they want to specify and IDEs still give information on what props are available.